docs/testing.md

```diff
@@ -0,0 +1,140 @@
+## Testing
+
+`tsaga` comes with its testing utilities.
+
+The utilities are similar to `redux-saga-test-plan` and modeled after how a saga would run in your application.
+
+### Sagas
+
+The assertions use the `expect` function from the `jest` testing framework.
+
+```typescript
+import { expectSaga, select, call, run, spawn } from 'tsaga';
+
+it('should run a saga', () => {
+  return (
+    expectSaga(watchForFetchUser)
+      // Setup a store with a reducer and an optional initial state
+      // This is optional
```

Comment:
As a first time reader I wonder what is now option. The state (as noted in the previous line) or the whole `withReducer`? (Which I don't is optional really, since then the final state helper wouldn't always make sense)